core/src/test/java/org/apache/iceberg/deletes/TestRoaringPositionBitmap.java

```diff
@@ -0,0 +1,323 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.deletes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+import org.apache.iceberg.relocated.com.google.common.collect.ImmutableSet;
+import org.apache.iceberg.relocated.com.google.common.io.Resources;
+import org.junit.jupiter.api.Test;
+
+public class TestRoaringPositionBitmap {
```

Comment:
I think this would be a good place to do some property based testing since we already have an implementation we think should work ("HashSet<Long>"). This way we can hit more than a select few positions and verify the behaviors pretty simply I think?

Reply:
Will add some fuzz testing.

Reply:
I added some fuzz tests and included seed information in the test arguments. If anything fails, we will have a way to reproduce the issue reliably.